doc/userguide/upgrade.rst

```diff
@@ -37,6 +37,10 @@ Major changes
 ~~~~~~~~~~~~~
 - Upgrade of PCRE1 to PCRE2. See :ref:`pcre-update-v1-to-v2` for more details.
 
+Security changes
+~~~~~~~~~~~~~~~~
+- Process creation by Suricata is now disabled by default, this can be configured with `security.limit-noproc` (for LUA scripts for instance)
```

Comment:
This is a bit terse, can you expand a bit?

Reply:
Expanding a bit.

Is the upgrade guide the right spot @jufajardini ?

Reply:
Answered in the following PR :P sorry for being slow on this one.